pkgs/applications/editors/emacs/elisp-packages/tree-sitter-langs/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, pkgs
+, symlinkJoin
+, fetchzip
+, melpaBuild
+, stdenv
+, fetchFromGitHub
+, writeText
+}:
+
+let
+  version = "0.10.14";
+  tree-sitter-grammars = stdenv.mkDerivation rec {
+    name = "tree-sitter-grammars";
+
+    inherit version;
+
+    src = fetchzip {
+      name = "tree-sitter-grammars-linux-${version}.tar.gz";
+      url = "https://github.com/emacs-tree-sitter/tree-sitter-langs/releases/download/${version}/${src.name}";
+      sha256 = "sha256-J8VplZWhyWN8ur74Ep0CTl4nPtESzfs2Gh6MxfY5Zqc=";
+      stripRoot = false;
+    };
+
+    installPhase = ''
+      install -d $out/langs/bin
+      echo -n $version > $out/langs/bin/BUNDLE-VERSION
+      install -m444 * $out/langs/bin
+    '';
+  };
+in melpaBuild rec {
```

nit: `rec` not needed here
```suggestion
in melpaBuild {
```